tools/releaser/lib/version_updater.dart

```diff
@@ -157,3 +161,33 @@ Future<bool> _updateReadmeVersions(CommandArguments args, Logger logger) async {
 
   return true;
 }
+
+Future<bool> _updateNativeSDKVersions(CommandArguments args, Logger logger) async {
+  final nativeSDKVersionsFile = File(path.join(args.packageRoot, 'NATIVE_SDK_VERSIONS.md'));
+  final newVersionEntry ='| ${args.version} | ${args.iOSRelease} | ${args.androidRelease} |';
+  final header = '| Flutter | iOS SDK | Android SDK |';
+  final separator = '|---------|---------|-------------|';
+
+  if (!nativeSDKVersionsFile.existsSync()) {
+    logger.warning('⚠️ NATIVE_SDK_VERSIONS.md does not exist, creating it now.');
+    await nativeSDKVersionsFile.writeAsString('''$header\n$separator\n$newVersionEntry''');
```

Comment:
This will be handled by dart-format but the triple quotes here aren't needed.

Reply:
Thanks for raising, changes at  [0fa8f1c](https://github.com/DataDog/dd-sdk-flutter/pull/737/commits/0fa8f1c2dcc0659dcc1bd0c4a54d31d240d5c598)